node/src/actors/chain_manager/mod.rs

```diff
@@ -887,17 +887,23 @@ pub fn process_validations(
     Ok(utxo_dif)
 }
 
+// This struct count the number of truths, lies and errors committed by an identity
+#[derive(Debug, Default, Clone, Eq, PartialEq)]
+struct RequestResult {
+    pub truths: u32,
+    pub lies: u32,
+    pub errors: u32,
+}
+
 #[derive(Debug, Default)]
 struct ReputationInfo {
     // Counter of "witnessing acts".
     // For every data request with a tally in this block, increment alpha_diff
     // by the number of witnesses specified in the data request.
```

Comment:
Unrelated to this PR, but this comment is outdated
```suggestion
    // by the number of reveals present in the tally.
```